<ui version="4.0" >
 <class>TTPreviewWidget</class>
 <widget class="QDialog" name="TTPreviewWidget" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>838</width>
    <height>573</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy vsizetype="MinimumExpanding" hsizetype="MinimumExpanding" >
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize" >
   <size>
    <width>660</width>
    <height>550</height>
   </size>
  </property>
  <property name="maximumSize" >
   <size>
    <width>6600000</width>
    <height>5500000</height>
   </size>
  </property>
  <property name="windowTitle" >
   <string>TTCut - Preview</string>
  </property>
  <layout class="QGridLayout" >
   <item row="1" column="0" >
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QComboBox" name="cbCutPreview" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="Minimum" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize" >
        <size>
         <width>220</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize" >
        <size>
         <width>220</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbPlay" >
       <property name="text" >
        <string>Play</string>
       </property>
       <property name="icon" >
        <iconset resource="previewwidget.qrc" >
         <normaloff>:/pixmaps/pixmaps/play_18.xpm</normaloff>:/pixmaps/pixmaps/play_18.xpm</iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>71</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pbExit" >
       <property name="text" >
        <string>Exit</string>
       </property>
       <property name="icon" >
        <iconset resource="previewwidget.qrc" >
         <normaloff>:/pixmaps/pixmaps/fileclose_18.xpm</normaloff>:/pixmaps/pixmaps/fileclose_18.xpm</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0" >
    <widget class="QFrame" name="videoFrame" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
       <horstretch>0</horstretch>
       <verstretch>6</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShape" >
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Sunken</enum>
     </property>
     <layout class="QGridLayout" >
      <property name="margin" >
       <number>2</number>
      </property>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="previewwidget.qrc" />
 </resources>
 <connections/>
</ui>
